lib/dealership.rb

```diff
@@ -0,0 +1,68 @@
+class Dealership
+  attr_accessor :inventory, :name, :location, :lot_value
+
+  def initialize(name, location)
+    @name = name
+    @location = location
+    @inventory = []
+    @lot_value = 0
+  end
+
+  def inventory_count
+    @inventory.length
+  end
+
+  def add_car(car)
+    @inventory << car
+  end
+
+  def has_inventory?
+    if inventory_count == 0
```

Comment:
Remember that our operators will return a boolean. So you would not need to use a `if/else` here. 

Also, if we are checking that a dealership has inventory wouldn't we want to check that our inventory count is anything greater than 0?
```
inventory_count > 0
```